### PR TITLE
Create release V0.29 RC 1

### PR DESCRIPTION
**Pull request by nithyatsu**
# Description

Create release candidate v0.29.0

## Auto-generated summary

copilot:all

### CODE REVIEW

File: docs/contributing/contributing-code/contributing-code-first-commit/contributing-azure-open-ai-related.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Configuring Azure Open AI
```
Comment (youngbupark) [inline docs/contributing/contributing-code/contributing-code-first-commit/contributing-azure-open-ai-related.md:1]: Can you please create a separate PR for this ?

**Comment by youngbupark** [comment]
Please do not use v0.29 as a branch because it could conflict with tag name later. use the different branch like this `<yourusername>/rel-0.29-rc1`

**Comment by youngbupark** [comment]
@willdavsmith Please review this PR.

**Comment by nithyatsu** [comment]
> @willdavsmith Please review this PR.

https://github.com/radius-project/radius/pull/6987 has the enw branch. Closing this PR.

**Pull request closed without merging**